### PR TITLE
Add new tool LINKS

### PR DESCRIPTION
**Pull request by TomHarrop**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [X] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

----

Adding the scaffolding tool [LINKS](https://github.com/bcgsc/LINKS)

### CODE REVIEW

File: tools/links/links.xml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    ]]></version_command>
+    <command detect_errors="exit_code"><![CDATA[
+
+        ## set up file paths
+
+        #set sequences_fn = 'sequences.' + $f.ext
+        ln -s '$f' '$sequences_fn'
+        &&
+        #set n = 1
+        #for $read in $reads
```
Comment (bgruening) [inline tools/links/links.xml:24]:
```suggestion
        #for $n, $read in enumerate($reads):
```

File: tools/links/links.xml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        #end for
+
+        ## RUN LINKS       
+
+        LINKS
+        -f '$sequences_fn'
+        -s reads.fof
+        -b links_output
+
+        #if $scaffolding.k:
```
Comment (bgruening) [inline tools/links/links.xml:40]: I think you need to cast them to str() since a user choice of 0 would evaluate to false

**Review by bgruening — COMMENTED** [review]
Very cool! Just two minor comments. Thanks @TomHarrop

File: tools/links/links.xml (hunk 1 above)
Comment (TomHarrop) [reply tools/links/links.xml]: Thanks, I always forget enumerate

File: tools/links/links.xml (hunk 2 above)
Comment (TomHarrop) [reply tools/links/links.xml]: I did not think of this! Thanks.

**Comment by TomHarrop** [comment]
Thanks for the help @bgruening, those are all fixed.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@TomHarrop gracias!

**Pull request merged**